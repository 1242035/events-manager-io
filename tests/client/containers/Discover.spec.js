import React from 'react';
import { shallow } from 'enzyme';
import ConnectedDiscover from '../../../client/src/containers/Discover';
import mockStore from '../mocks/storeMock';

describe('<Discover />', () => {
  let wrapper;
  let store;

  beforeEach(() => {
    store = mockStore();
    wrapper = shallow(<ConnectedDiscover store={store} />);
  });

  it('should render dumb component', () => {
    expect(wrapper.length).toEqual(1);
  });

  it('Should maintain existing snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
