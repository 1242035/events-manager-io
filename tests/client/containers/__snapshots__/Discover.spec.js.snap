// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Discover /> Should maintain existing snapshot 1`] = `undefined`;
